tests/test_goniometer.py

```diff
@@ -216,11 +231,42 @@ def test_goniometer_from_phil():
 
     assert g1.get_rotation_axis() == (1, 0, 0)
 
+
+def test_single_axis_goniometer_using_axis_and_axes_from_phil():
```

Comment:
Slightly more verbose test name:
```suggestion
def test_single_axis_goniometer_using_axis_and_axes_from_phil_raises_error():
```